cmake/Helpers.cmake

```diff
@@ -247,11 +247,24 @@ function(check_python_module module)
   endif()
 
   if(RET_MODULE_TEST)
-    message(STATUS "Installing module ${module} in Python venv")
-    execute_process(
-      COMMAND
-      ${Python_EXECUTABLE} -m pip install ${module}
-    )
+    if(ENABLE_AUTO_DOWNLOAD)
+      message(STATUS "Installing module ${module_name} in Python venv")
+      execute_process(
+        COMMAND
+        ${Python_EXECUTABLE} -m pip install ${module_name}
+        RESULT_VARIABLE PYTHON_MODULE_INSTALL_CMD_EXIT_CODE
+      )
+      if(PYTHON_MODULE_INSTALL_CMD_EXIT_CODE)
+        message(FATAL_ERROR "Could not install ${module_name} in venv")
+      endif()
+    else()
+      message(FATAL_ERROR
+        "Could not find module ${module_name} for Python "
+        "version ${Python_VERSION_MAJOR}.${Python_VERSION_MINOR}. "
+        "Make sure to install ${module_name} for this Python version "
+        "via \n`${Python_EXECUTABLE} -m pip install ${module_name}'.\n"
+        "Note: You need to have pip installed for this Python version.")
```

Comment:
Maybe mention that --auto-download can also be specified to automatically install the package.

Reply:
Done.